src/main/java/com/github/packageurl/PackageURL.java

```diff
@@ -727,24 +756,47 @@ public int hashCode() {
      *
      * @since 1.0.0
      */
-    public static class StandardTypes {
+    public static final class StandardTypes {
+        public static final String ALPM = "alpm";
+        public static final String APK = "apk";
         public static final String BITBUCKET = "bitbucket";
+        public static final String BITNAMI = "bitnami";
+        public static final String COCOAPODS = "cocoapods";
         public static final String CARGO = "cargo";
         public static final String COMPOSER = "composer";
-        public static final String DEBIAN = "deb";
+        public static final String CONAN = "conan";
+        public static final String CONDA = "conda";
+        public static final String CPAN = "cpan";
+        public static final String CRAN = "cran";
+        public static final String DEB = "deb";
         public static final String DOCKER = "docker";
         public static final String GEM = "gem";
         public static final String GENERIC = "generic";
         public static final String GITHUB = "github";
         public static final String GOLANG = "golang";
+        public static final String HACKAGE = "hackage";
         public static final String HEX = "hex";
+        public static final String HUGGINGFACE = "huggingface";
+        public static final String LUAROCKS = "luarocks";
         public static final String MAVEN = "maven";
+        public static final String MLFLOW = "mlflow";
         public static final String NPM = "npm";
         public static final String NUGET = "nuget";
+        public static final String QPKG = "qpkg";
+        public static final String OCI = "oci";
+        public static final String PUB = "pub";
         public static final String PYPI = "pypi";
         public static final String RPM = "rpm";
-        public static final String NIXPKGS = "nixpkgs";
-        public static final String HACKAGE = "hackage";
+        public static final String SWID = "swid";
+        public static final String SWIFT = "swift";
+        // FIXME: Remove this since it should be named DEB
+        public static final String DEBIAN = "deb";
+        // FIXME: Remove this since it should be named NIX and it is not a standard type
+        public static final String NIXPKGS = "nix";
```

Comment:
https://github.com/package-url/purl-spec/blob/8040ff0be50f0c5b1986b1a0947bd539f5405fc4/PURL-TYPES.rst?plain=1#L620

Unless all candidates should also be added, but then the `Standard` in the name of the class isn't exactly accurate.

Reply:
I'm okay with adding all the candidates for now. There are plans to codify the list of types in a machine-readable format so that all implementations can refer to the list and use the same types.